docs/index.md

```diff
@@ -32,16 +32,16 @@ and methods into the `pytket.extensions` namespace.
 .. autosummary::
     :nosignatures:
 
-    IBMQBackend
-    IBMQEmulatorBackend
-    AerBackend
-    AerStateBackend
-    AerUnitaryBackend
-    AerDensityMatrixBackend
+    backends.ibm.IBMQBackend
```

Comment:
Could we prefix these with `~` to make the table look slightly nicer?

<img width="770" height="363" alt="Screenshot 2025-07-24 at 15 01 02" src="https://github.com/user-attachments/assets/4a6048eb-4c0f-4237-afb3-ce90f1c96018" />